oauth/1.1/modules/ROOT/pages/oauth-module-troubleshooting.adoc

```diff
@@ -0,0 +1,41 @@
+= Troubleshooting OAuth Module - Mule 4
+
+To troubleshoot OAuth module, become familiar with the information about enabling verbose logging and interpreting commonly throw messages.
+
+== Enable Verbose Logging
+
+To get a better understanding of why an application's interaction with OAuth module is failing, temporarily enable verbose logging for the module.
+
+Remember to always disable the enhanced verbosity after troubleshooting, because it affects your Mule application performance.
+
+To enable verbose logging in the configuration file:
+
+. Access Anypoint Studio and navigate to the *Package Explorer* view.
+. Open your application's project name.
+. Open the `src/main/resources` path folder.
+. Open the `log4j2.xml` file inside the folder.
+. Add an `<AsyncLogger>` tag inside the `<Loggers>` tag:
++
+[source,xml,linenums]
+----
+			<Loggers>
+				...
+				<AsyncLogger name="org.mule.extension.oauth2" level="DEBUG"/>
+				...
+			</Loggers>
+----
+[start=6]
+. Save your application changes.
+. Click the project name in *Package Explorer* and then click *Run* > *Run As* > *Mule Application*.
+
+
+== Understand Common Throws
+
+Here is a list of common thrown messages and how to interpret them:
+
+* HTTP:SECURITY
+
+ Occurs when there is an error related to an HTTP request.
```

Comment:
So periods or now periods in these lists is something to discuss in our connectors meeting. In this case, Kim would have said not to use a period because this is not a full sentence, but if you think there will be throw descriptions where we do need periods, we should probably treat this like we treat the Reference guide tables and always use them. I am in favor of doing that.